development-standards.md

```diff
@@ -382,14 +337,17 @@ ALL branch merges should only be done via a Pull Request. Please fill in all rel
 
 Items marked with * can be deleted if you are not working from an issue.
 
-You should also assign reviewers to your pr and assign yourself. Merges to `develop` should have at least 2 reviewers. One of those reviewers must be [Alex Foxleigh](https://github.com/foxleigh81).
+You should also assign reviewers to your PR and assign yourself. Merges to `develop` should have at least 1 reviewer but the more people who review a PR the better. One of those reviewers must be [Alex Foxleigh](https://github.com/foxleigh81) or [Ian Mayo](https://github.com/IanMayo).
 
 A label should also be applied indicating what was done. These things are easy to overlook but a PR should not be approved without them. To help developers remember to do this, a checklist is included in a PR which must be completed truthfully. PR's which do not conform to this will be rejected.
 
 Once a pull request has been approved and all of the checks have passed, the originator of the PR should perform the merge. The branch attached to that PR should be automatically deleted but if not please delete it manually.
 
 If you are working in a child branch then please ensure that the pull request merges to the parent branch and not `develop` and add a note about that to the 'developer notes' section.
 
+If a PR is listed as 'Draft' then please ignore it entirely until the developer marks it as 'ready for review', this will avoid confusion.
```

Comment:
I think it may be worth being explicit with the `Draft` status of PRs.  Devs in other projects are unaware of it, and put `WIP` in the title. So, maybe either include the exact GitHub naming of the feature, or include a screenshot.